samples/cpp/speculative_decoding_lm/speculative_decoding_lm.cpp

```diff
@@ -116,34 +30,25 @@ int main(int argc, char* argv[]) try {
     };
 
     ov::genai::SchedulerConfig scheduler_config;
-    // batch size
-    scheduler_config.max_num_batched_tokens = use_prefix ? 256 : 32;
-    // cache params
-    scheduler_config.num_kv_blocks = 364;
+    scheduler_config.cache_size = 5;
     scheduler_config.block_size = get_default_block_size(device);
-    // mode - vLLM or dynamic_split_fuse
-    scheduler_config.dynamic_split_fuse = dynamic_split_fuse;
-    // vLLM specific params
-    scheduler_config.max_num_seqs = 2;
-    scheduler_config.enable_prefix_caching = use_prefix;
 
     // It's possible to construct a Tokenizer from a different path.
     // If the Tokenizer isn't specified, it's loaded from the same folder.
```

Comment:
let's drop comments about tokenizers as they are not meaningful for the SD sample